proto/minder/v1/minder.proto

```diff
@@ -470,6 +526,7 @@ service ProfileService {
 
         option (rpc_options) = {
             auth_scope: OBJECT_OWNER_PROJECT
+            relation: RELATION_PROFILE_GET
```

Comment:
I'd say we should have a dedicated relation for this. e.g. `profile_status_get`. Even if it just gets added to the existing roles, it's handy to have such granularity.

Reply:
Are you suggesting we add it to the open fga model? Or just to the proto? I'm fine with either

Reply:
The suggestion is for bothboth. To have a `profile_status_get` relation in the FGA model and add it here as well. what do you think?

Reply:
Works for me, I'll update